src/test/java/com/fasterxml/jackson/databind/deser/std/UUIDDeserializerTest.java

```diff
@@ -0,0 +1,58 @@
+package com.fasterxml.jackson.databind.deser.std;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.Base64;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+
+class UUIDDeserializerTest {
+  @Test
+  void testCanDeserializeUUIDFromString() throws IOException {
+    final UUID uuid = UUID.randomUUID();
```

Comment:
Instead of using new generated random UUIDs, it'd be better to generate one randomly but include as constant.
This to avoid any possibility of having non-reproducible fails (at expense of not covering many combinations).

It is unfortunate JDK does not offer a way to see generator for random-number variant, for testing (but perhaps they were worried about devs using that in production and end up with collisions), as otherwise we'd use that.
JUG of course can do that but perhaps it's better not to add it as a (test) dependency,.

Reply:
ok, I can do that 👍